### PR TITLE
adding color and font style

### PR DESCRIPTION
**Pull request by Matchimaw**
- Adding color to scss file
- Config rails for font 
- Adding animal crossing font for a header

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import 'color';
```
Comment (dovakeen118) [inline app/assets/stylesheets/main.scss:1]: LOVE. THIS.

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -12,3 +12,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+
+Rails.application.config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
```
Comment (dovakeen118) [inline config/initializers/assets.rb:16]: Wondering if this line could be combined with the one on line 9 for node modules?

**Review by dovakeen118 — COMMENTED** [review]
I'm not sure if you tried combining the lines in the `assets.rb` file - let me know!

**Comment by Matchimaw** [comment]
- Combined  line 16 with line 9 in config/assets.rb file

**Pull request merged**